frontend/apps/marketing/src/components/snapshots/labSnapshot/__tests__/LabSnapshot.test.tsx

```diff
@@ -36,15 +36,29 @@ describe('LabSnapshot component', () => {
     });
 
     const labSnapshot = screen.getByTitle(title);
+
     expect(labSnapshot).toBeVisible();
+    expect(labSnapshot).toHaveTextContent(
+      'Ages: 1, 1st' +
```

Comment:
Looks much better! Another nit is to use template literals rather than concatenating so we don't have all those `+`.